source/dbo/Stored Procedures/Extracts/uspPopulateForestExtract.sql

```diff
@@ -35,6 +35,7 @@ BEGIN
 			,Postcode
 			,Sex
 			,Hospital
+			,HospitalTBServiceCode
```

Comment:
Needs to be TB Service Name, the codes don't mean anything to anyone.

Reply:
Ah I didn't realise that, I thought they were more publicly know. Switched to using TB service names